### ISSUE TITLE
NTR, Triggered by an abusive adult, Munchausen syndrome

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5808]
**Preferred term label:**
Triggered by an abusive adult

**Synonyms**
Munchausen syndrome by proxy

**Definition (free text, please give PubMed ID)**
Applies to a sign or symptom that is induced by an abusive adult (usually a parent)
Comment: Munchausen syndrome by proxy is not a pediatric condition, but the outward manifestation of an adult psychiatric disorder.
 PMID: 29938745 ; DOI: 10.7417/T.2018.2067 

**Parent term (use hpo.jax.org/app)**
"Triggered by"
https://hpo.jax.org/app/browse/term/HP:0025204

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
lmu chrapp, @katrinknoflach, @mgriese

**Comment by GrieseAG** [comment]
unclear about the noted parent term or if there is a better one for the inclusion of the desired HPO term

**Comment by pnrobinson** [comment]
added as HP:0033032
